tests/integration/olp-cpp-sdk-authentication/TokenEndpointTest.cpp

```diff
@@ -30,7 +30,10 @@
 #include <olp/core/http/NetworkConstants.h>
 #include <olp/core/http/NetworkUtils.h>
 #include <olp/core/porting/make_unique.h>
+#include <olp/core/porting/warning_disable.h>
```

Comment:
Do we still need to disable warnings in this file, after most of the deprecations were removed?

Reply:
I will enable warnings here)